### PR TITLE
refunds application demo deployment

### PR DESCRIPTION
**Pull request by Jennings-HMCTS**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-refunds-api
+spec:
+  releaseName: ccpay-refunds-api
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
```
Comment (vijayrajagopalan-hmcts) [inline k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml:9]: Repository is incorrect. It should point to charts in Github.

Example: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml#L8-L10

File: k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    name: ccpay-refunds-api
+    version: 0.1.4
+  values:
```
Comment (vijayrajagopalan-hmcts) [inline k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml:12]: Values section is incorrect.

Refer: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/rd/rd-caseworker-ref-api/rd-caseworker-ref-api.yaml#L11-L15

The image used in this config is correct.

File: k8s/namespaces/fees-pay/ccpay-refunds-app/demo.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-refunds-api
+spec:
+  values:
+    java:
+      environment:
```
Comment (vijayrajagopalan-hmcts) [inline k8s/namespaces/fees-pay/ccpay-refunds-app/demo.yaml:8]: You can add a dummy variable, which can be used to restart pods when required.

File: k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml (hunk 2 above)
Comment (Jennings-HMCTS) [reply k8s/namespaces/fees-pay/ccpay-refunds-app/ccpay-refunds-app.yaml:12]: changes done

File: k8s/namespaces/fees-pay/ccpay-refunds-app/demo.yaml (hunk 3 above)
Comment (Jennings-HMCTS) [reply k8s/namespaces/fees-pay/ccpay-refunds-app/demo.yaml]: dummy variable added

**Review by adusumillipraveen — APPROVED** [review]

**Review by vijayrajagopalan-hmcts — APPROVED** [review]

**Pull request merged**